core/vm/evm.go

```diff
@@ -25,12 +25,13 @@ import (
 
 	"github.com/scroll-tech/go-ethereum/common"
 	"github.com/scroll-tech/go-ethereum/crypto"
+	"github.com/scroll-tech/go-ethereum/crypto/codehash"
 	"github.com/scroll-tech/go-ethereum/params"
 )
 
 // emptyCodeHash is used by create to ensure deployment is disallowed to already
 // deployed contract addresses (relevant after the account abstraction).
-var emptyCodeHash = crypto.Keccak256Hash(nil)
+var emptyCodeHash = codehash.EmptyCodeHash
```

Comment:
why some are `codehash.EmptyCodeHash.Bytes()` whereas some are `codehash.EmptyCodeHash`?

Reply:
Some packages work with the type `common.Hash`, while others work with `[]byte`, but the data is the same for these two.